src/help/zaphelp/contents/start/concepts/ddc.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Data Driven Content
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Data Driven Content</H1>
+<p>
+Data driven content is type of <a href="structmods.html">Structural Modifier</a>
+which identifies URL paths that represent data.<br>
+<p>
+In 'traditional' web applications the structure of the application is typically defined by the URL paths
+and the data is contained in the URL parameters and POST data.<br>
+URLs like:
+<ul>
+<li>https://www.example.com/app/aaa?ddd=eee
+<li>https://www.example.com/app/aaa?ddd=fff
+<li>https://www.example.com/app/bbb?ddd=eee
+</ul>
+are represented in the <a href="../../ui/tabs/sites.html">Sites tab</a> as two 'nodes' in the tree:
+<ul>
+<li>Sites
+	<ul>
+	<li>https://www.example.com
+		<ul>
+		<li>app
+			<ul>
+			<li>GET:aaa(ddd)
+			<li>GET:bbb(ddd)
+			</ul>
+		</ul>
+	</ul>
+</ul>
+The Sites tree is very important as it reflects ZAP's understanding of the application structure.<br>
+If it is not a good representation of the structure then ZAP will not be able to attack the application effectively.<br>
+<br>
+Some applications include data in URL paths.<br>
+For example: 
+<ul>
+<li>https://www.example.com/app/company1/aaa?ddd=eee
+<li>https://www.example.com/app/company2/aaa?ddd=fff
+<li>https://www.example.com/app/company3/aaa?ddd=ggg
+</ul>
+These 3 URLs represent the same page but with different data, but by default ZAP will represent them as three separate nodes:
+<ul>
+<li>Sites
+	<ul>
+	<li>https://www.example.com
+		<ul>
+		<li>app
+			<ul>
+			<li>company1
+				<ul>
+				<li>GET:aaa(ddd)
+				</ul>
+			<li>company2
+				<ul>
+				<li>GET:aaa(ddd)
+				</ul>
+			<li>company3
+				<ul>
+				<li>GET:aaa(ddd)
+				</ul>
+			</ul>
+		</ul>
+	</ul>
+</ul>
+This is a problem because ZAP will now attack all 3 pages when it only needs to attack one of them.<br>
+In this case attacking the same page 3 times is not a big problem, but if you have hundreds or thousands 
+of pages like this will significantly increase the time it takes to scan the application.<br>
+<br>
+In ZAP terms the 'company' nodes are 'data driven content' - URL parameters that contain
+data instead of representing part of the application structure.<br>
+You can define data driven content by adding the application to a <a href="contexts.html">Context</a>
+and then configuring them via the <a href="../../ui/tabs/sites.html">Sites tab</a> 'Flag as Context/<i>Context Name</i> Data driven node' right click menu item<br>
```

Comment:
The separation of menu and menu item could/should have a space character before and after the slash, it should improve the readability.
We should also define (and normalise) which separator should be used (for example, in some places it's used "->").